### PR TITLE
Updated mode for  bi-encoder

### PR DESCRIPTION
**Pull request by AshishMahendra**
Please the details of the evaluation
image

### CODE REVIEW

File: jskit/encoders/utils/config.cfg [hunk 1]
```diff
@@ -18,8 +18,8 @@ gpu = 0
 basepath = log_output
 
 [MODEL_PARAMETERS]
-architecture = bi
+architecture = bi (obselete)
```
Comment (ypkang) [inline jskit/encoders/utils/config.cfg]: Why not just remove it if it's not being used anymore?

File: jskit/encoders/utils/config.cfg [hunk 2]
```diff
@@ -1,14 +1,14 @@
 [TRAIN_PARAMETERS]
 max_contexts_length = 128
-MAX_CANDIDATE_LENGTH = 64
-train_batch_size = 32
-eval_batch_size = 2
-MAX_HISTORY = 4
+max_candidate_length = 64
+train_batch_size = 8
+eval_batch_size = 1
+max_history = 4
```
Comment (ypkang) [inline jskit/encoders/utils/config.cfg:6]: are you sure this should be lower case? if you look at `train.py` its all reading using upper case? can you confirm and make sure train/infer work?

**Review by ypkang — CHANGES_REQUESTED** [review]

File: jskit/encoders/utils/config.cfg (hunk 2 above)
Comment (AshishMahendra) [reply jskit/encoders/utils/config.cfg:6]: Yes, it does work

**Review by ypkang — APPROVED** [review]
LGTM

**Pull request merged**